swarm/network/stream/delivery.go

```diff
@@ -80,7 +80,7 @@ func (d *Delivery) handleRetrieveRequestMsg(ctx context.Context, sp *Peer, req *
 
 	var cancel func()
 	// TODO: do something with this hardcoded timeout, maybe use TTL in the future
-	ctx = context.WithValue(ctx, "peer", sp.ID().String())
+	ctx = context.WithValue(ctx, "peer", sp.bzzPeer.ID().String())
```

Comment:
This is changing behaviour on `master`. One of the things that I am addressing in `simple-fetchers` is the use of `context` as a container for values that change the logic of execution.

I am not sure if it is okay to change `enode.ID` to `bzzAddr` here.